testdata/p4_16_samples_outputs/issue242-midend.p4

```diff
@@ -76,21 +76,33 @@ control Eg(inout Headers hdrs, inout Metadata meta, inout standard_metadata_t st
     bool tmp_2;
     bit<32> tmp_3;
     bit<32> tmp_4;
+    bool cond;
```

Comment:
We do have tests with actions calling actions, but the inlining pass is run before this one and inlines all such calls.